### ISSUE TITLE
External Addressability (Ingress): How to expose Solr under a sub-path?

### ISSUE THREAD

**Issue by ngc4579** [apache/solr-operator#601]
Using the the `SolrClouds` CR and `Ingress` external addressability method, is there a way to expose Solr under a sub-path, i.e. something like `https://my.custom.domain/some/path/to/solr`?

**Comment by HoustonPutman** [comment]
Hmmm the solr operator doesn't support that currently. Have you set Solr to listen to a different host context other than "/solr"? The solr operator assumes this path quite a lot, so I'm not sure it would really work.

**Comment by ngc4579** [comment]
Yeah, I already thought so... thank you for clarification. I've found an alternative approach, disabling external addressability via the operator CR and instead deploying SolrClouds in an umbrella chart with an additional (custom) Ingress resource. Works so far (evaluation system), apart from Keycloak (OIDC) integration...
